chain/cosmos/cosmos_chain.go

```diff
@@ -36,27 +36,6 @@ type CosmosChain struct {
 	ChainNodes    ChainNodes
 }
 
-func NewCosmosChainConfig(name string,
```

Comment:
Dead code. Probably worth running a linter to find and delete unused code at some future point.

Reply:
this was used in the `test custom` command of the previous CLI. It would be nice to make this functional in the matrix file